COMMUNICATION.md

```diff
@@ -0,0 +1,23 @@
+
+And while we all deserve to collaborate without fear of overt harassment and personal attacks, that is a very low bar! This document is more aspirational and has fewer "teeth" (i.e., it's not about "banning" people), but it's also very important: it describes **how we wish to communicate with each other** and **how we aspire to create an environment that cultivates trust**, even in the face of inevitable technical and organizational disagreement.
+
+Finally, this is **a living document** and should be adapted over time as the OpenTelemetry project evolves. At any time, there are certainly errors of omissions or even simple clarifications we can make to the wording here, and any and all are welcome to propose such changes.
+
+Now, without further ado:
+
+## Communication guidelines
+
+What follows are an unordered list of principles that guide our communication with each other – both active community members as well as those who may just be "passing through" to file an issue or suggest an improvement.
+
+* **Critique ideas, not people:** It's fine and expected to disagree about technology and project direction. Even when those disagreements are significant or frustrating, though, it's not fine to criticize the participants. In fact, it's usually a sign that there's no strong technical argument being made. (More specifically, see [this post](https://developers.redhat.com/blog/2019/07/08/10-tips-for-reviewing-code-you-dont-like/) for tips about code reviews in particular)
+* **Be welcoming:** By nature, OpenTelemetry touches a lot of other projects and will thus have a larger-than-normal ratio of casual committers and passers-by. For this reason, it's especially important to be welcoming to newcomers. This means thanking them for their interest and contributions, and being patient with their initial misunderstandings. In fact, their points of confusion are a valuable signal for documentation gaps.
+* **Try to keep conversations in the open:** Where possible, try to have conversations in async formats like GitHub issues, PRs, and emails. If resolution requires a call or videoconference, try to make a recording and notes available and discoverable to keep our community as accessible as possible across distant timezones.
+* **Strive for consensus, but don't require it:** Of course it's great when we can get everyone to agree. And certainly everyone should be – and should feel – heard. As a rule of thumb, if there is a lone dissenter (or three) in any given discussion, their points should have been clearly addressed: not just with a simplistic "Sorry, I disagree," but with a principled counterargument. (Note that the way we ultimately make decisions in spite of consensus is not just a communication issue, but a matter of policy and process; as such, that is out of scope for this document)
```

Comment:
The parenthetical here feels a little contorted. Is the intent to say "Note that project governance and the rules for final decision making are covered elsewhere"?

Reply:
I did something...

Reply:
LGTM